packages/@aws-cdk/aws-codepipeline-actions/test/codestar-connections/codestar-connections-source-action.test.ts

```diff
@@ -148,26 +148,140 @@ describe('CodeStar Connections source Action', () => {
       ],
     });
 
+  });
+
+  test('exposes variables', () => {
+    const stack = new Stack();
+    createBitBucketAndCodeBuildPipeline(stack, {});
```

Comment:
Let's make the second argument of `createBitBucketAndCodeBuildPipeline ()` optional, this will allow us to write:

```suggestion
    createBitBucketAndCodeBuildPipeline(stack);
```